components/collapsible/collapsible.test.js

```diff
@@ -88,6 +87,16 @@ describe('Dialtone vue Collapsible Component Tests', function () {
     });
   });
 
+  describe('When scoped slot is provided', function () {
+    it('should render the scoped slot', function () {
+      const anchor = '<button data-qa="anchor-element">click me</button>';
+      slots = { anchor };
+      _mountWrapper();
+      const anchorSlotContainer = wrapper.find('[data-qa="anchor-element"]');
```

Comment:
We put all of our `.find` in `_setChildWrappers`. You can just recall it if needed.